modules/installation-and-upgrade/pages/container-deployment/suma/server-migration-suma.adoc

```diff
@@ -26,16 +26,18 @@ Self trusted GPG keys are not migrated.
 GPG keys that are trusted in the RPM database only are not migrated.
 Thus synchronizing channels with [command]``spacewalk-repo-sync`` can fail.
 
-The administrator must migrate these keys manually from the 4.3 installation **after migration**:
+The administrator must migrate these keys manually from the 4.3 installation to the container host before the actual server migration.
```

Comment:
I am ignorant about that part (the GPG keys).

Reply:
I changed this again. I should have thought twice about it.